providers/flagd/src/test/java/dev/openfeature/contrib/providers/flagd/resolver/process/TestUtils.java

```diff
@@ -15,21 +18,23 @@ public class TestUtils {
     public static final String INVALID_CFG = "flagConfigurations/invalid-configuration.json";
     public static final String UPDATABLE_FILE = "flagConfigurations/updatableFlags.json";
 
-
     public static String getFlagsFromResource(final String file) throws IOException {
-        final URL url = FlagParser.class.getClassLoader().getResource(file);
-        if (url == null) {
-            throw new IllegalStateException(String.format("Resource %s not found", file));
-        } else {
-            return new String(Files.readAllBytes(Paths.get(url.getPath())));
```

Comment:
`url.getPath` creates a path string representation, that is not accepted by `Paths.get(String)` on Windows (e.g. `/C:/sample.csv`).

See [this issue](https://bugs.openjdk.org/browse/JDK-8197918).